src/abstractdataframe/subset.jl

```diff
@@ -77,7 +92,8 @@ Each argument passed in `args` can be either a single column selector or a
 described for [`select`](@ref).
 
 Note that as opposed to [`filter`](@ref) the `subset` function works on whole
-columns (or all rows in groups for `GroupedDataFrame`).
+columns and always makes a subset of rows (or all rows in groups for
```

Comment:
What does "makes a subset of rows" mean? Also it's not mentioned for `subset!`.

Reply:
it is intended to mean that selection happens row-wise always (not whole groups) - this was the original reason why this PR is made. In particular this is a crucial difference between `subset` and `filter` for `GroupedDataFrame` (as the latter works per group). Could you please propose a better wording to convey this message?

Reply:
OK. Then maybe this?
```
works on whole columns (and selects rows within groups for `GroupedDataFrame`
rather than whole groups).
```

And I'd mention that the result should be a vector of Boolean in the first paragraph as that's really an essential information.

Reply:
OK - added